Description: filenameforall calls bad iodev with insufficent scratch
 This was a regression introduced with commit
 65a9046ded8e9edd5d33bc812a9e94ae29607a1e
 .
 I hadn't realised the cleanup procedure was expected to be called
 in the case of both the success *and* error conditions.
Origin: upstream, http://git.ghostscript.com/?p=ghostpdl.git;h=9565f4c
Author: Ken Sharp <ken.sharp@artifex.com>
Forwarded: yes
Bug: http://bugs.ghostscript.com/show_bug.cgi?id=699813
Last-Update: 2018-10-18
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/psi/zfile.c
+++ b/psi/zfile.c
@@ -438,7 +438,6 @@
             esp -= 5;               /* pop proc, pfen, devlen, iodev , mark */
             return o_pop_estack;
         } else if (code > len) {      /* overran string */
-            esp -= 5;               /* pop proc, pfen, devlen, iodev , mark */
             return_error(gs_error_rangecheck);
         }
         else if (iodev != iodev_default(imemory)
